README.md

```diff
@@ -361,14 +458,18 @@ available in geckodriver by viewing the help message:
 
 ## Building
 
-geckodriver is written in [Rust](https://www.rust-lang.org/)
-and you need the [Rust toolchain](https://rustup.rs/) to compile it.
+geckodriver is written in [Rust](https://www.rust-lang.org/),
+a new systems programming language from the [Mozilla](https://www.mozilla.org/en-US/),
+that is fast, prevents segfaults, and guarantees thread safety.
```

Comment:
I stole this directly from the website, but I’m OK to leave it out.

It now says: “geckodriver is written in Rust, a systems programming language from Mozilla. In order to build […]”